specification/cat/thread_pool/CatThreadPoolRequest.ts

```diff
@@ -55,9 +55,9 @@ export interface Request extends CatRequestBase {
     /**
      * List of columns to appear in the response. Supports simple wildcards.
      */
-    h?: Names
+    h?: CatThreadPoolColumn
     /**
-     * List of columns that determine how the table should be sorted.
+     * A comma-separated list of column names or aliases that determines the sort order.
```

Comment:
For what it's worth, the clients all accept an actual list, and will create the comma-separated value themselves. This is why it was worded like that. But the new wording is fine and more generic :+1: